build_integration_test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+cli="./faas-cli"
+if [ $(uname) == 'Darwin' ] ; then
+    cli="./faas-cli-darwin"
+elif [ -f "/etc/debian_version" ] ; then
+    if [ $(dpkg --print-architecture) == 'armhf' ] ; then
+        cli="./faas-cli-armhf"
+    elif [ $(dpkg --print-architecture) == 'arm64' ] ; then
+        cli="./faas-cli-arm64"
+    fi
+fi
+
+eval $cli new $FUNCTION --lang python3
```

Comment:
Now that we've got things parameterized let's define a bash function for this whole block.